sllurp/verb/capabilities.py

```diff
@@ -0,0 +1,71 @@
+"""Reset command.
+"""
+
+import logging
+import pprint
+from sllurp.llrp import LLRPReaderConfig, LLRPReaderClient, LLRPReaderState
+from sllurp.log import get_logger
+
+logger = get_logger(__name__)
+
+
+def shutdown(reader, state):
+    host, port = reader.get_peername()
+    logger.info('Shutting down reader %s:%d', host, port)
+    reader.disconnect()
+
+def main(args):
+    if not args.host:
+        logger.info('No readers specified.')
+        return 0
```

Comment:
```suggestion
        return 1
```

Returning something other than 0 will indicate clearly to the user that the program did not exit normally.